prebid-mobile/pbm-api/android/pbm-targeting-android.md

```diff
@@ -745,31 +745,37 @@ Parameters:
 
 ## Arbitrary OpenRTB
 
-(requires SDK v2.2.1)
-
-While there are many specific methods for adding data to the request detailed in
-this document, OpenRTB is big and it moves quickly. To cover scenarios not already covered by an existing method,
-Prebid SDK Provides a way for app publishers to customize most ORTB fields in the partial bid request that Prebid Mobile sends to the Prebid Server. The customization comes in the form of the ortbConfig parameter that takes a JSON String as input. The JSON string must follow the [OpenRTB structure](https://github.com/InteractiveAdvertisingBureau/openrtb2.x/blob/main/2.6.md) -- it will be merged with the current JSON of the bid request. If you choose to input extra data using the ortbConfig parameter, please extensively test your requests sent to Prebid Server.
-
-There are certain protected fields such as regs, device, geo, ext.gdpr, ext.us_privacy, and ext.consent which cannot be changed.
-
-Here's how to specify cross-impression data:
-
-```kotlin
-//global invocation
-adUnitConfiguration?.ortbConfig = "{\"ext\":{\"prebid\":{\"debug\":1,\"trace\":\"verbose\"}}}"
+(requires SDK v2.4.0)
```

Comment:
I like this interface much better @YuriyVelichkoPI , but I know that we have customers using the old adUnitConfiguration.ortbConfig interface that probably won't upgrade to 2.4.0 soon. I think we need to keep the 2.2.1 method in the docs for a while.

Reply:
Thanks for catching this. We'll update PRs to keep the current behavior as is. The current methods will be marked as deprecated, so publishers will be aware of the necessity to use new ones.

Reply:
thank you.